### DISCUSSION TITLE
has_many field

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5820] [Q&A]
I have two tables for example : one is users and the second is addresses.
I added field with type has many in blueprint and add in the addresses model belongsToMany function and the data inserted in third table is call : users_addresses but when i make edit the dropdown with mutli select not be selected the items from the database, it is empty but the data aleady in the database. How to fix this

**Comment by ryanmitchell** [comment]
Is this a runway specific issue? You would be better asking there: https://github.com/doublethreedigital/runway
